src/components/Charts&Tables/BudgetChart.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
+
+import { Bar } from 'react-chartjs-2'
+
+
+class BudgetChart extends React.Component {
+    constructor(props) {
+        super(props)
+        this.state = {
+            cashAccountData: []
+        }
+    }
+
+    static getDerivedStateFromProps(props, state) {
+
+        function getCashAccountsData(account) {
+            const data = {
+                labels: ['balance', 'budget' ,'expense'],
+                datasets: [
+                    {
+                        label: account.title,
+                        data: [account.balance, account.limit ,account.expenses],
+                        backgroundColor: ['#64c93c', '#c93c41', '#3cc9aa'],
```

Comment:
why do we have styles added in components, we can just use CSS files with the same name as component and just import the CSS file here and use the classes instead